exporter/exporter.go

```diff
@@ -386,6 +354,42 @@ func (e *Exporter) Handler() http.Handler {
 	})
 }
 
+// GetRequestOpts makes exporter.Opts structure from request filters and default options.
+func GetRequestOpts(filters []string, defaultOpts *Opts) Opts {
+	requestOpts := Opts{}
+
+	if len(filters) == 0 {
+		requestOpts = *defaultOpts
+	}
+
+	for _, filter := range filters {
+		switch filter {
+		case "diagnosticdata":
+			requestOpts.EnableDiagnosticData = true
+		case "replicasetstatus":
+			requestOpts.EnableReplicasetStatus = true
+		case "dbstats":
+			requestOpts.EnableDBStats = true
+		case "topmetrics":
+			requestOpts.EnableTopMetrics = true
+		case "currentopmetrics":
+			requestOpts.EnableCurrentopMetrics = true
+		case "indexstats":
+			requestOpts.EnableIndexStats = true
+		case "collstats":
+			requestOpts.EnableCollStats = true
+		case "profile":
+			requestOpts.EnableProfile = true
+		case "shards":
+			requestOpts.EnableShards = true
+		case "pbm":
+			requestOpts.EnablePBMMetrics = true
```

Comment:
seems FCV was missed during the merge:
```suggestion
		case "pbm":
			requestOpts.EnablePBMMetrics = true
		case "fcv":
			requestOpts.EnableFCV = true
		
```

Reply:
Omg, how did that happen.. fixed